Formula/php-solr.rb

```diff
@@ -0,0 +1,15 @@
+require_relative "../lib/php_pecl_formula"
+
+class PhpSolr < PhpPeclFormula
+  extension_dsl "PHP extension for Apache Solr"
+
+  url "https://pecl.php.net/get/solr-2.5.1.tgz"
+  sha256 "2b41601e74fe371e6525579637a8ddff270e58a0ff636e7f56df8919d6e7777d"
+  license "PHP-3.0"
```

Comment:
It's actually `PHP-3.01`

Reply:
indeed, I just copied it from xdebug package without paying detailed attention